app/timetables/models.py

```diff
@@ -175,3 +175,16 @@ def __str__(self):
 
     class Meta:
         unique_together = ('timetable', 'cycle_day', 'meal', 'meal_option')
+
+
+class Vendor(SlugifyMixin, TimestampMixin):
+    """Model representing food service-provider"""
+
+    name = models.CharField(max_length=255)
```

Comment:
Yes, it should be required and it is required. Django does not have _required_ as one of its supported field options. However, if a field is desired not to be required, field options _null_ or _blank_ would be set to True.